### ISSUE TITLE
Errors running compiled 4D v18r3

### ISSUE THREAD

**Issue by truegold** [miyako/4d-component-classic-query-editor#6]
Hi Miyako,

I downloaded and compiled as component (obviously placed in our component folder next to the app) to use in application upgraded to 4D v18r3 running on Mac OS 10.13.6. Receiving the attached error.

	C_OBJECT($params)
	$table_ptr:=$1
	
	  // $u161215 note:
	  //  Query button enabled if "selection" set to False
	  //  If property "repeat" set to False then clicking on a search will close the editor
	
	OB SET($params;"tableNumber";Table($table_ptr);"repeat";False;"developer";False;"language";"en";"selection";False)
	
	CLASSIC_QUERY ($params)

If I run component as source code then there are no errors.

I'm not sure what needs to fix?

John...

[search_01.pdf](https://github.com/miyako/4d-component-classic-query-editor/files/4974243/search_01.pdf)

**Comment by miyako** [comment]
First step is the click the "details" button. 
There is no need to hide this part when reporting an issue,
other than to annoy the developer. 😫

<img width="640" alt="details" src="https://user-images.githubusercontent.com/1725068/89094998-a1ef6580-d404-11ea-9e35-a76a3ae643e3.png">

That said, screenshots are for non-programmers.
Developers should click "copy" and paste the actual error stack.

```
Error when executing the method "ON_LOAD" at line number 3

Attempting to retype by using a pointer.

Error code: -20007 (4DRT)
Attempting to retype by using a pointer.
component: '4DRT'
task -6, name: 'P_1'
```

**Comment by miyako** [comment]
I think it is a typo

```4d
If (TEXT_Try_to_load (Form.path;$JSON))
	Editor_QUERY_LOAD ($JSON->)
End if 
```

**Comment by truegold** [comment]
Hi Miyako,

Sorry didn’t mean to make life harder. When I took the screen shot there was no copy button. And when I expanded the details it looked like the same message - although I can understand it did include the error number.



Also it’s been a habit to always have my clients send me a screenshot. Sorry about that!
> Closed #6 <https://github.com/miyako/4d-component-classic-query-editor/issues/6>.
> 
I have made the change and that appears to have worked. We will test further next week.

Please understand I didn’t mean to offend you or be sloppy in my bug reporting processes.

Appreciate your help,
John…
